definitions/v1/schema.json

```diff
@@ -609,7 +609,7 @@
                 "^((title|description)\\|(append))?$": {
                     "$ref": "#/definitions/SelectorBlock"
                 },
-                "^((category)\\|(noappend))?$": {
+                "^((category)\\|(noappend|append))?$": {
```

Comment:
This isn’t supported which is why it’s not in, appending/union is the default value and doesn’t need called out specifically so why use it in definitions? You could use `add`, `union`, `smash` and get same behavior. The schema keeps this tight.

https://github.com/Prowlarr/Prowlarr/blob/5ee95e3cc29d1307192320eb82b5a8f1287f00d6/src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannParser.cs#L459


https://github.com/Jackett/Jackett/blob/7868b74f35efd2756f981fea1fb0ffcde54d51a9/src/Jackett.Common/Indexers/CardigannIndexer.cs#L1966

Reply:
@ilike2burnthing see Q's comments

Reply:
It was used in the .yml files for bmet and funkytorrents to address https://github.com/Jackett/Jackett/issues/11917. I think `|append` was just used for the sake of consistency.